api/pom.xml

```diff
@@ -36,6 +36,11 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-jaxb-adapters</artifactId>
+      <version>1.0.6-SNAPSHOT</version>
```

Comment:
`hawkular-accounts-jaxb-adapters` seems to be a module in the present tree, right? I know about the problems with ${project.version} but is this not a place where those problems are harmless?